doc/links.txt

```diff
@@ -1,33 +1,39 @@
-.. _`Socket API`: https://pubs.opengroup.org/onlinepubs/9699919799/functions/socket.html
+.. _`nrfx`: https://github.com/NordicSemiconductor/nrfx/
 
-.. _`Beej's Guide to Network Programming`: https://beej.us/guide/bgnet/
+.. ### Links to Infocenter
 
 .. _`AT Commands Reference Guide`: https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/intro.html
 
 .. _`Credential storage management %CMNG`: https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/cmng.html
 
 .. _`LTE Link Monitor`: https://infocenter.nordicsemi.com/topic/ug_link_monitor/UG/link_monitor/lm_certificate_manager.html
 
+.. _`nRF IEEE 802.15.4 radio driver documentation`: https://infocenter.nordicsemi.com/topic/struct_drivers/struct/radio_driver_latest.html
```

Comment:
This link is going to be outdated.
When 1.3.0 relase is out we will have Radio Driver 2.0.0-1.prealpha released there. The doc should point to 2.0.x version. We will be commiting new doc as the Radio Driver evolves to 2.1.0 2.2.0, but in version 1.3.0 of nRF Connect SDK link will not point to 2.0.0-1.prealpha anymore as released but to newer releases which are not part of 1.3.0 nRF Connect SDK.

Reply:
Updated link to v2.0.0-1.prealpha. We can discuss link versioning after NCS 1.3 release.